### PR TITLE
Features/#002 fix subnetworks and scenario parameters

### LINKED ISSUE #2: adjust fix_ehv_subnetworks for historic sq scenario
considering a new osm dataset - the osmtgmod data has to be checked and than this script has to be adjusted:
https://github.com/openego/powerd-data/blob/features/status_quo_pipeline/src/egon/data/datasets/fix_ehv_subnetworks.py

### PR DESCRIPTION
**Pull request by CarlosEpia**
Fixes #2 and #26.

## Before merging into `dev`-branch, please make sure that

- [ ] the `CHANGELOG.rst` was updated.
- [X] new and adjusted code is formated using `black` and `isort`.
- [ ] the `Dataset`-version is updated when existing datasets are adjusted.
- [ ] the branch was merged into the
      `continuous-integration/run-everything-over-the-weekend`-branch.
- [x] the workflow is running successful in `test mode`.
- [ ] the workflow is running successful in `Everything` mode.

### CODE REVIEW

File: src/egon/data/datasets/fix_ehv_subnetworks.py [hunk 1]
```diff
@@ -230,57 +225,87 @@ def fix_subnetworks(scn_name):
...
-            8.402976949446648,
-            53.2371468322213,
+            12,
+            853318030996972,  # Umspannwerk Plottling
+            48,
+            76508569478315,
+            12,
+            769768646403532,  # Gänsdorf UW
+            48,
+            80533685376445,
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/fix_ehv_subnetworks.py]: Looks a bit strange to me. Is the format really correct?

File: src/egon/data/datasets/fix_ehv_subnetworks.py [hunk 2]
```diff
@@ -230,57 +225,87 @@ def fix_subnetworks(scn_name):
...
-            11.4295305,
-            50.9115176,
+            11,
+            4295305,  # Ottstedt
+            50,
+            9115176,
+            11,
+            4299277,  # Ottstedt
+            50,
+            9114496,
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/fix_ehv_subnetworks.py]: See comment above

File: src/egon/data/datasets/fix_ehv_subnetworks.py [hunk 3]
```diff
@@ -230,57 +225,87 @@ def fix_subnetworks(scn_name):
...
+        #     # Line north south
+        #     11.4298432,
+        #     50.9117467,
+        #     # Line east
+        #     11.4295305,
+        #     50.9115176,
+        #     380,
+        #     scn_name,
+        #     3,
+        # )
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/fix_ehv_subnetworks.py]: I'm not sure if it is the best option to comment the lines that aren't needed for this osm dataset. Right now, it is not that clear why the lines are there but not executed. 
I would either delete them or implement an if-statement or sth. similar that checks the osm dataset date. Or - as a minimal solution you could at least add a comment describing why you did that.

File: src/egon/data/datasets/scenario_parameters/parameters.py [hunk 4]
```diff
@@ -129,6 +129,25 @@ def global_settings(scenario):
...
+            "co2_costs": 76.5,  # [EUR/t_CO2]
+            "co2_emissions": {  # Netzentwicklungsplan Strom 2035, Version 2021, 1. Entwurf, p. 40, table 8
+                "waste": 0.165,  # [t_CO2/MW_th]
+                "lignite": 0.393,  # [t_CO2/MW_th]
+                "gas": 0.201,  # [t_CO2/MW_th]
+                "nuclear": 0.0,  # [t_CO2/MW_th]
+                "oil": 0.288,  # [t_CO2/MW_th]
+                "coal": 0.335,  # [t_CO2/MW_th]
+                "other_non_renewable": 0.268,  # [t_CO2/MW_th]
+            },
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/scenario_parameters/parameters.py]: I think you do not need the fuel_costs, co2_costs and co2_emissions for this PR

File: src/egon/data/datasets/scenario_parameters/parameters.py [hunk 5]
```diff
@@ -472,7 +491,177 @@ def electricity(scenario):
...
+            + global_settings(scenario)["co2_costs"]
+            * global_settings(scenario)["co2_emissions"]["coal"],
+            "nuclear": global_settings(scenario)["fuel_costs"]["nuclear"]
+            + read_costs(costs, "nuclear", "VOM"),
+            "biomass": global_settings(scenario)["fuel_costs"]["biomass"]
+            + read_costs(costs, "biomass CHP", "VOM"),
+            "wind_offshore": read_costs(costs, "offwind", "VOM"),
+            "wind_onshore": read_costs(costs, "onwind", "VOM"),
+            "solar": read_costs(costs, "solar", "VOM"),
+        }
```
Comment (ClaraBuettner) [inline src/egon/data/datasets/scenario_parameters/parameters.py]: I guess these parameters["marginal_cost"] are not needed in this PR

File: src/egon/data/datasets/fix_ehv_subnetworks.py (hunk 1 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/fix_ehv_subnetworks.py]: There was an inconsistency with "." and ",". Corrected in a4dd369747b662e5d2eb5b4ff3bc84f6097a0f4d

File: src/egon/data/datasets/fix_ehv_subnetworks.py (hunk 2 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/fix_ehv_subnetworks.py]: There was an inconsistency with "." and ",". Corrected in a4dd369747b662e5d2eb5b4ff3bc84f6097a0f4d

File: src/egon/data/datasets/fix_ehv_subnetworks.py (hunk 3 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/fix_ehv_subnetworks.py]: The commented lines were deleted in a4dd369747b662e5d2eb5b4ff3bc84f6097a0f4d

File: src/egon/data/datasets/scenario_parameters/parameters.py (hunk 4 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/scenario_parameters/parameters.py]: Fixed in bbb6aaeff58eb52e627c2be87bec658e03396b95

File: src/egon/data/datasets/scenario_parameters/parameters.py (hunk 5 above)
Comment (CarlosEpia) [reply src/egon/data/datasets/scenario_parameters/parameters.py]: Fixed in bbb6aaeff58eb52e627c2be87bec658e03396b95

**Review by ClaraBuettner — APPROVED** [review]

**Pull request merged**